src/appendices/phenotype.md

```diff
@@ -0,0 +1,250 @@
+# Tabular phenotypic data guidelines
+
+This appendix is a collection of guidelines and examples utilizing these guidelines for creating well-organized aggregated tabular phenotypic data.
+
+## Guidelines
+
+The following guidelines are all **RECOMMENDED** when preparing tabular phenotypic data like the participants file, sessions file, demographics file, or phenotypic and assessment data. The language below uses REQUIRED, MUST, and others to imply these are the requirements for these **RECOMMENDED** guidelines.
+
+### 1. Aggregation
+
+Tabular phenotypic data files are prepared as one pair of data dictionary JavaScript Object Notation (JSON) file and data tab-separated value (TSV) file. Aggregation refers to the contents of the TSV file, collecting all participant data into one TSV per tabular phenotypic file.
```

Comment:
```suggestion
Phenotypic data MUST be prepared as one pair of a tabular file in tab-separated value (TSV) format and a corresponding data dictionary in  JavaScript Object Notation (JSON) format. Aggregation refers to the contents of the TSV file, collecting all participant data into one TSV per tabular phenotypic file.
```
This is essentially a change from MAY -> MUST in [https://bids-specification.readthedocs.io/en/stable/common-principles.html#tabular-files](https://www.google.com/url?q=https://bids-specification.readthedocs.io/en/stable/common-principles.html%23tabular-files&sa=D&source=docs&ust=1741875485953254&usg=AOvVaw0oThgLuHy_0wqO4JT_Lqhq) right?

Reply:
I am thinking if the part about "MUST have sidecar/data dictionary" shouldn't get its own section above because it is an important difference b/w the guidelines and the base spec, and I don't think it has much to do with "Aggregation" otherwise